sources/dynamodb/schema.go

```diff
@@ -247,10 +281,9 @@ type statItem struct {
 	Count int64
 }
 
-func inferDataTypes(stats map[string]map[string]int64, rows int64, s *schema.Table) {
-	if s.ColDefs == nil {
```

Comment:
Is there any flow where previous colDefs will be required ?

Reply:
inferDataTypes is the only one that sets coldefs for dynamodb and its only called once so its always nil.